deploy/crds/openebs.io_blockdeviceclaims_crd.yaml

```diff
@@ -0,0 +1,159 @@
+      openAPIV3Schema:
+        description: BlockDeviceClaim is the Schema for the BlockDeviceClaim CR
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: DeviceClaimSpec defines the request details for a BlockDevice
+            properties:
+              blockDeviceName:
+                description: BlockDeviceName is the reference to the block-device
+                  backing this claim
+                type: string
+              blockDeviceNodeAttributes:
+                description: BlockDeviceNodeAttributes is the attributes on the node
+                  from which a BD should be selected for this claim. It can include
+                  nodename, failure domain etc.
+                properties:
+                  hostName:
+                    description: HostName represents the hostname of the Kubernetes
+                      node resource where the BD should be present
+                    type: string
+                  nodeName:
+                    description: NodeName represents the name of the Kubernetes node
+                      resource where the BD should be present
+                    type: string
+                type: object
+              deviceClaimDetails:
+                description: Details of the device to be claimed
+                properties:
+                  allowPartition:
+                    description: AllowPartition represents whether to claim a full
+                      block device or a device that is a partition
+                    type: boolean
+                  blockVolumeMode:
+                    description: 'BlockVolumeMode represents whether to claim a device
+                      in Block mode or Filesystem mode. These are use cases of BlockVolumeMode:
+                      1) Not specified: VolumeMode check will not be effective 2)
+                      VolumeModeBlock: BD should not have any filesystem or mountpoint
+                      3) VolumeModeFileSystem: BD should have a filesystem and mountpoint.
+                      If DeviceFormat is    specified then the format should match
+                      with the FSType in BD'
```

Comment:
Is the `description` generated from comments ?

Reply:
Yes, that's why I removed the //TODO comments.